supreme/src/androidMain/kotlin/at/asitplus/signum/supreme/os/AndroidKeyStoreProvider.kt

```diff
@@ -290,7 +292,7 @@ sealed class AndroidKeystoreSigner private constructor(
     internal val jcaPrivateKey: PrivateKey,
     final override val alias: String,
     val keyInfo: KeyInfo,
-    private val config: AndroidSignerConfiguration,
+    internal val config: AndroidSignerConfiguration,
```

Comment:
i don't like making this internal, and it only needs to be internal as a consequence of the key agreement being an extension function, which it should not be